jenkins-x.yml

```diff
@@ -0,0 +1,15 @@
+buildPack: ml-python-training
+pipelineConfig:
+
+  pipelines:
+    overrides:
+      - pipeline: pullRequest
+        stage: build
+        name: testing
+        step:
+          sh: source /root/.bashrc && python3 -m pip install -r requirements.txt && pytest -v
```

Comment:
No need for this as everything is done on master - jx actually won't accept PR builds